playwright-e2e/lib/component/dsm/filters/sections/search/search.ts

```diff
@@ -56,6 +59,11 @@ export class Search {
     textValue && !exactMatch && (await this.setExactMatch(columnName, true));
   }
 
+  public async radioBtn(columnName: string, { radioButtonValue, additionalFilters}: Partial<RadioButtonConfig>): Promise<void> {
```

Comment:
nit: Change to `radiobutton` in order to keep name consistent with `widget/radiobutton` class.